deployment/compose.yaml

```diff
@@ -1,18 +1,38 @@
 services:
-  etl-insitu:
-    container_name: etl-insitu
+  db:
+    container_name: mongo
+    image: mongo:latest
+    restart: always
+    ports:
+      - 27017:27017
+    volumes:
+      - database:/data/db
+  insitu-etl:
     image: ghcr.io/ecmwfcode4earth/vairify/vairify-etl-insitu
+    container_name: etl-insitu
+    depends_on:
+      - db
+    build:
+      context: ./air-quality-backend/
+      dockerfile: Dockerfile.insitu
     environment:
-      - MONGO_DB_URI=${MONGO_DB_URI}
+      - MONGO_DB_URI=mongodb://mongo:27017/
```

Comment:
I think currently the UI has a username/password in it. Not sure what best practice is here but will this work as is?

Reply:
So we can add a username and password to the database if we want to. I decided against it for now, mainly because the access to the box is controlled. By UI do you mean MongoCompass? or the dashboard?

Reply:
I meant URI sorry! In the env file. - If we don't have username/password in the URI will we be able to connect in the application?

Reply:
Just realised you probably meant URI, I think the same answer applies